nativelib/src/main/scala/scala/scalanative/regex/Parser.scala

```diff
@@ -1496,9 +1501,13 @@ object Parser {
                 max == -2 || max > 1000 || max >= 0 && min > max) {
               // Numbers were negative or too big,
               // or max is present and min > max.
-              throw new PatternSyntaxException(ERR_INVALID_REPEAT_SIZE,
-                                               t.from(start),
-                                               0)
+              throw new PatternSyntaxException(
+                ERR_INVALID_REPEAT_SIZE,
+//                                               t.from(start),
+//                                               0)
```

Comment:
There are stray comments here, which should be removed I assume.

Reply:
Thank you.

Because I was not able, within available time & budge overrun, to create a test case for
that exception, I reverted to the previous git code.

There is a wide spread low lying issue in the scalanative/regex port with the reported
error positions being 0 instead of something approximately correct.  I have been fixing
them as I can provoke them in test cases.

 The original 0 positions were done in the port to get the whole of the matter working 
but I knew at the time that I would be chasing them for the rest of my professional lifetime
(and, most probably, into the beyond).